### ISSUE TITLE
for...of loop (64 iterations) with console.log statement causing sandbox to freeze and crash (NOT INFINTE LOOP)

### ISSUE THREAD

**Issue by d0rf47** [codesandbox/codesandbox-client#6294]
# 🐛 bug report

## Preflight Checklist

- [ ] I have read the [Contributing Guidelines][contributing] for this project.
- [ ] I agree to follow the [Code of Conduct][code_of_conduct] that this project
      adheres to.
- [ ] I have searched the issue tracker for an issue that matches the one I want
      to file, without success.

## Description of the problem  
 Trying to run a console.log() statement within a for of loop consisting of 64 iterations causes the code sandbox to freeze and crash
## How has this issue affected you? What are you trying to accomplish?
trying to view html element id's inside for...of loop (64 elements total)
### To Reproduce
open the code sandbox below and uncomment the for..of loop inside js/game.js initGame()

### Your Environment

| Software         | Name/Version |
| ---------------- | ------------ |
| Сodesandbox      |         unsure     |
| Browser          |        chrome      |
| Operating System |        windows      |

[code_of_conduct]: https://github.com/codesandbox/codesandbox-client/blob/master/CODE_OF_CONDUCT.md
[contributing]: https://github.com/codesandbox/codesandbox-client/blob/master/CONTRIBUTING.md

**Comment by CompuIves** [comment]
Oh wow... I can reproduce this. That's so weird, because the application runs under a different context than the main page, and this shouldn't be too complicated to run. Thanks a lot for reporting this!

My only hunch at this could be that our console renderer cannot render the `id`s for some reason. Are these `id`s strings? Because if they're strings I also can't imagine that that is the problem....

**Comment by CompuIves** [comment]
Oh, interesting, in the browser console it's empty objects... So it could definitely be that our console has trouble with rendering.
![image](https://user-images.githubusercontent.com/587016/142386626-2e6d0a31-f52f-424b-af7a-00ad35ca11a7.png)

**Comment by d0rf47** [comment]
yeah thats what i was figuring too. I had a similar issue as issue #884 which i believe i was doing the same tutorial and was running an infinite loop, but this is definitely a unique case, I actually had to export my files and work locally cause it become too difficult to work in the sandbox and view my log statements which i use a lot for debugging and learning new things.

thanks for looking into it.

**Comment by d0rf47** [comment]
> Oh wow... I can reproduce this. That's so weird, because the application runs under a different context than the main page, and this shouldn't be too complicated to run. Thanks a lot for reporting this!
> 
> My only hunch at this could be that our console renderer cannot render the `id`s for some reason. Are these `id`s strings? Because if they're strings I also can't imagine that that is the problem....

yes they are simply html id's strings
`<div id="a1" class="tile"></div>` is an example of the id. there are 64 tiles (its a chess set) when i run locally there was 0 issue
